code/game/machinery/cloning.dm

```diff
@@ -379,6 +379,7 @@
 
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
+		to_chat(occupant, "<span class='warning'><b>You remember nothing after you've blacked out and you do not remember who or what events killed you, however, you can have faint recollection of what led up to it.</b>") //Skyrat change - reminds you about the blackout policy
```

Comment:
Maybe even worth a bigger text class on here?

Reply:
It's <b>old though

Reply:
yeah I guess, prob fine